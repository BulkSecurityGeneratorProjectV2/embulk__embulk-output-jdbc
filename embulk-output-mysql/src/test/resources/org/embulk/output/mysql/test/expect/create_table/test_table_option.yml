table: test1
mode: insert
create_table_option: 'auto_increment=100'
column_options:
  id: {type: 'int auto_increment primary key'}
