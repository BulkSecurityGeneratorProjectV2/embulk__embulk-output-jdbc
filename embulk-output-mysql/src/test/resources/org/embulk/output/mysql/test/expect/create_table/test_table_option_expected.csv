100,a
101,b
102,c
